plugins/apitracing/Readme.md

```diff
@@ -1,2 +1,90 @@
 [![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=gdatasoftwareag_apitracing&metric=ncloc)](https://sonarcloud.io/summary/new_code?id=gdatasoftwareag_apitracing)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=gdatasoftwareag_apitracing&metric=coverage)](https://sonarcloud.io/summary/new_code?id=gdatasoftwareag_apitracing)
+
+# ApiTracing Plugin
+
+This plugin hooks functions of specific processes and extracts the function parameters.
```

Comment:
Better: "This plugin allows the user to hook API functions in specified processes which are either running already or being started during the analysis run."